PresentData/PresentMonTraceConsumer.cpp

```diff
@@ -2218,7 +2215,7 @@ void PMTraceConsumer::HandleProcessEvent(EVENT_RECORD* pEventRecord)
             };
             mMetadata.GetEventData(pEventRecord, desc, _countof(desc));
             event.ProcessId     = desc[0].GetData<uint32_t>();
-            event.ImageFileName = desc[1].GetData<std::string>();
+            event.ImageFileName = desc[1].GetData<std::wstring>();
             event.IsStartEvent  = true;
```

Comment:
AFAIK the underlying data for this event is always char, and GetData<T>() doesn't do any conversion, it just accesses the event data as if it were a T.  So, we need to leave it as GetData<string>() and then convert here.

In debug build GetData() should be asserting with this change, though you'll only hit this event when processing from an ETL file (--etl_file).